pkgs/tools/misc/automirror/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, fetchFromGitHub, bash, git, ronn}:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "automirror";
+  version = "49";
+
+  src = fetchFromGitHub {
+    owner = "schlomo";
+    repo = "automirror";
+    rev = "v${version}";
+    sha256 = "1syyf7dcm8fbyw31cpgmacg80h7pg036dayaaf0svvdsk0hqlsch";
+  };
+
+  patchPhase = "sed -i s#/usr##g Makefile";
+
+  buildInputs = [ git ronn ];
+
+  installPhase = ''
+    make install DESTDIR=$out
```

Comment:
Please use `installFlags` to set `DESTDIR=$out` instead of overwriting `installPhase`.

Reply:
updated - thanks for the hint :+1: